examples/js/loaders/FBXLoader.js

```diff
@@ -386,6 +386,21 @@
 		texture.wrapS = valueU === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
 		texture.wrapT = valueV === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
 
+		if ( 'Scaling' in textureNode.properties ) {
+
+			values = textureNode.properties.Scaling.value;
+
+			if ( typeof values === 'string' ) {
+
+				values = parseFloatArray( values );
+
+			}
```

Comment:
It really just needs to be a single line - this pattern is used throughout the loader without the `typeof === 'string'` check. The `var` is important too

    var values = parseFloatArray( textureNode.properties.Scaling.value );

Reply:
`function parseFloatArray( string )` expects a string but the `values` could be either a string or an array depending on the FBX exported / modelling software. Blender produces an array while Modo produces a string.

Reply:
`function parseFloatArray( string )` expects a string but the `values` could be either a string or an array depending on the FBX exporter settings. For example with Modo if it's a binary FBX it will be parsed as an array and if it's a plain text FBX it will be parsed as a string. Here it's parsed to a float array only if it's a string.

`var` is important, that should be fixed.

Reply:
Hmm, you are correct. Sorry! 

I wonder why this hasn't come up before, seems unlikely that it's just this property that is affected.

Reply:
Actually, I guess there are lots of places throughout the loader where it is checking things like: 

```
case 1:
case '1': 
```

It just hasn't come up for an array before. It seems like the right thing to do here would be to go through the binary parser and do the type checking there, rather lots of places throughout the loader. 

In the meantime this is a good solution 👍